ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/SchemaInfo.java

```diff
@@ -75,4 +79,30 @@ public boolean equals(final Object other) {
   public int hashCode() {
     return Objects.hash(type, fields, memberSchema);
   }
+
+  private static final Map<SqlBaseType, Function<SchemaInfo, String>> TO_TYPE_STRING =
+      ImmutableMap.<SqlBaseType, Function<SchemaInfo, String>>builder()
+          .put(SqlBaseType.STRING, si -> "VARCHAR(STRING)")
```

Comment:
Can we just use `STRING`?

Reply:
sure - I didn't change anything here from what it was previously, but I'll do that :)

Reply:
actually - this would be a relatively intrusive change to docs and unit tests. I will leave it as is